packages/cli/web/index.js

```diff
@@ -32,26 +23,59 @@ const start = ({ state }) => {
 
         // process refresh token
         if (token) {
-          // write creds
-          fs.writeFileSync(path.join(credsPath, "creds.json"), JSON.stringify(token))
+          if (!ciServer) {
+            // write creds
+            fs.writeFileSync(path.join(credsPath, "creds.json"), JSON.stringify(token))
```

Comment:
No need to touch the user session in the CI case. Assuming the cred.json file already existed

Reply:
In the CI case, the Creds will not be rewrite.

Reply:
I think its ok, will check with elon and change otherwise